app/assets/javascripts/govuk_publishing_components/load-analytics.js

```diff
@@ -71,38 +73,119 @@ window.GOVUK.loadAnalytics = {
     }
   },
 
+  loadGa4Vars: function () {
+    // If we recognise the environment, load the GA4 vars
+    if (this.ga4EnvironmentVariables[this.environment]) {
+      window.GOVUK.analyticsGa4 = window.GOVUK.analyticsGa4 || {}
+      window.GOVUK.analyticsGa4.vars = window.GOVUK.analyticsGa4.vars || {}
+      window.GOVUK.analyticsGa4.vars.id = 'GTM-MG7HG5W'
+      window.GOVUK.analyticsGa4.vars.auth = this.ga4EnvironmentVariables[this.environment].auth
```

Comment:
@andysellick Good idea - I may leave it for now as I think that will break the tests since they currently rely on being able to override those values. For example, if you wanted to test disabling GA4 on production works, in our tests we can currently do `window.GOVUK.loadAnalytics.ga4EnvironmentVariables['production'].initialiseGa4 = false` and `spyOn` is a `jasmine-jquery` thing so I have been trying to avoid using that.